arabic/practice.md

```diff
@@ -0,0 +1 @@
+تطبيق ما تعلمته من هذا الدرس في أعمالي سيكون الخطوة المقبلة لترسيخ المعلومات و الإستفادة
```

Comment:
@AmeniSassi, it looks like your fix was good, but now the file you moved exists in both places. You'll need to delete the old one using `git rm`.